backstop/tests/header.html

```diff
@@ -0,0 +1,971 @@
+            type="button"
+          >
+            <div class="iui-thumbnail-area">
+              <div class="iui-thumbnail"></div>
+            </div>
+            <span class="iui-label">
+              <div class="iui-name">Model Beta</div>
+              <div class="iui-id">0x0123456789</div>
+            </span>
+            <svg
+              viewBox="0 0 16 16"
+              xmlns="http://www.w3.org/2000/svg"
+              aria-hidden="true"
+              class="iui-icon"
+            >
+              <path d="M4.807 6h6.395a.28.28 0 01.24.443L8.27 9.9a.34.34 0 01-.481 0L4.566 6.443A.27.27 0 014.806 6z" />
+            </svg>
+          </button>
+        </div>
+
+        <div class="iui-center">
+
+        </div>
+
+        <div class="iui-right">
+          <!-- Notifications -->
+          <button
+            class="iui-button iui-borderless"
+            type="button"
+            aria-label="Notifications"
+            aria-expanded="false"
+          >
+            <!-- Use notification.svg from generic-icons package. -->
+            <svg
+              viewBox="0 0 16 16"
+              xmlns="http://www.w3.org/2000/svg"
+              aria-hidden="true"
+              class="iui-icon"
+            >
+              <path d="m7.5 14c.9 0 1.5-.4 1.5-1h-3c0 .6.7 1 1.5 1" />
+              <path
+                d="m11.7 10.4v-3.3c0-1.9-1.4-3.4-3.2-3.8v-.4c0-.4-.5-.9-1-.9-.6 0-1 .5-1 .9v.4c-1.9.5-3.2 2-3.2 3.8v3.3l-1.3 1.2v.4h11v-.4l-1.3-1.2"
+              />
+            </svg>
+          </button>
+
+          <!-- Help -->
+          <button
+            class="iui-button iui-borderless"
+            type="button"
+            aria-label="Help"
+            aria-expanded="false"
+          >
+            <!-- Use help-circular-hollow.svg from generic-icons package. -->
+            <svg
+              viewBox="0 0 16 16"
+              xmlns="http://www.w3.org/2000/svg"
+              aria-hidden="true"
+              class="iui-icon"
+            >
+              <path
+                d="m10.015 6.34611a2.51955 2.51955 0 0 1 -.93 1.79 1.99113 1.99113 0 0 0 -.78992 1.5h-1.11008a2.92148 2.92148 0 0 1 1.12-2.29c.78686-.67739.62127-1.96-.38-1.96-.63324.02175-.74.69627-.74 1.28h-1.2a2.8677 2.8677 0 0 1 .68-2.09 2.04145 2.04145 0 0 1 3.35 1.77zm-1.5196 4.80527a.74526.74526 0 1 1 -.74526-.74526.74525.74525 0 0 1 .74526.74526zm-.4954-9.65027a6.49889 6.49889 0 1 1 -6.4989 6.49889 6.50625 6.50625 0 0 1 6.4989-6.49889m0-1.5a7.99889 7.99889 0 1 0 7.9989 7.99889 7.99889 7.99889 0 0 0 -7.9989-7.99889z"
+              />
+            </svg>
+          </button>
+
+          <!-- Profile dropdown button -->
+          <button
+            class="iui-button iui-dropdown iui-borderless"
+            type="button"
+            aria-label="My account"
+            aria-expanded="false"
+          >
+            <span
+              class="iui-user-icons-medium"
```

Comment:
Missing `iui-user-icon` class so the demo animation actually animate the user icon

```suggestion
              class="iui-user-icons-medium iui-user-icon"
```